### PR TITLE
docs: Add Korean translation

### PR DESCRIPTION
**Pull request by preco21**
Hi!

I have translated AVA docs into Korean based on [this](https://github.com/sindresorhus/ava/commit/71404c23302d825095659c70cb9a1b08251697ad) commit.

This is first draft of the docs, so I will update remain docs later.
Please review the docs!

### CODE REVIEW

**Comment by sindresorhus** [comment]
@preco21 Awesome! ✨  Do you know anyone that could review?

**Comment by preco21** [comment]
@sindresorhus Sorry for late response. I have invited my friends to review the docs. Also, I have tweeted on my twitter too. Maybe someone gives wonderful helps for reviewing the docs. :)

File: ko_KR/readme.md [hunk 1]
```diff
@@ -0,0 +1,971 @@
...
+});
+
+test.afterEach.cb(t => {
+	setTimeout(t.end);
+});
+```
+
+참고로 `beforeEach`와 `afterEach` 그냥 테스트의 이전과 이후에 각각 실행되며 기본적으로 테스트를 동시에 실행시킵니다. 만약 각 테스트를 위해 전역 상태를 설정해야 할 필요가 있는 경우 ([예시](https://github.com/sindresorhus/ava/issues/560)와 같이 `console.log`를 감시한다거나), 테스트가 확실하게 [직렬화로 실행](#테스트를-직렬로-실행하기)하는지 확인해야 합니다.
+
+참고로 AVA는 각 테스트 파일을 분리된 프로세스에서 실행합니다. 따라서 `이후`-훅이 바로 프로세스가 종료되기 전에 실행되는 이상 따로 전역 상태를 정리하지 않아도 됩니다.
```
Comment (Riyeon) [inline ko_KR/readme.md]: Change `이후`-훅 to `after`-훅 will be better for consistency

**Comment by Riyeon** [comment]
LGTM!

File: ko_KR/readme.md (hunk 1 above)
Comment (preco21) [inline ko_KR/readme.md]: Thanks! Have updated.

**Comment by forresst** [comment]
@preco21 Great works !
I will review this week that the format is correct. (Not the korean of course, i'm french).

File: ko_KR/readme.md [hunk 2]
```diff
@@ -0,0 +1,971 @@
+___
+**역주**
+
+이 문서는 [readme.md](https://github.com/sindresorhus/ava/blob/master/readme.md)의 한국어 번역입니다. [이곳](https://github.com/sindresorhus/ava/commits/master)에서 AVA의 master 브랜치와 이 문서의 차이를 확인할 수 있습니다. (만약 차이가 없다면 문서가 최신 버전임을 의미합니다)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace
`(https://github.com/sindresorhus/ava/commits/master)`
By
`(https://github.com/sindresorhus/ava/commits/71404c23302d825095659c70cb9a1b08251697ad...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9)`

File: ko_KR/readme.md [hunk 3]
```diff
@@ -0,0 +1,971 @@
...
+- [팀](#팀)
+
+## 왜 AVA인가요?
+
+- 작고 빠릅니다
+- 간단한 테스트 문법
+- 테스트를 동시적으로 실행
+- 작은 테스트를 작성하도록 유도
+- 내재적인 전역 변수 없음
+- [각 테스트 파일마다 고립된 환경](#process-isolation)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#process-isolation)`
By:
`(#프로세스-고립)`

File: ko_KR/readme.md [hunk 4]
```diff
@@ -0,0 +1,971 @@
...
+- [ES2015로 테스트 작성](#es2015-support)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#es2015-support)`
By:
`(#es2015-지원)`

File: ko_KR/readme.md [hunk 5]
```diff
@@ -0,0 +1,971 @@
...
+- [Promise 지원](#promise-support)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#promise-support)`
By:
`(#promise-지원)`

File: ko_KR/readme.md [hunk 6]
```diff
@@ -0,0 +1,971 @@
...
+- [생성기 함수 지원](#generator-function-support)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#generator-function-support)`
By:
`(#생성기-함수-지원)`

File: ko_KR/readme.md [hunk 7]
```diff
@@ -0,0 +1,971 @@
...
+- [비동기 함수 지원](#async-function-support)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#async-function-support)`
By:
`(#비동기-함수-지원)`

File: ko_KR/readme.md [hunk 8]
```diff
@@ -0,0 +1,971 @@
...
+- [Observable 지원](#observable-support)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#observable-support)`
By:
`(#observable-지원)`

File: ko_KR/readme.md [hunk 9]
```diff
@@ -0,0 +1,971 @@
...
+- [강화된 가정 메시지](#enhanced-assertion-messages)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#enhanced-assertion-messages)`
By:
`(#강화된-가정-메시지)`

File: ko_KR/readme.md [hunk 10]
```diff
@@ -0,0 +1,971 @@
...
+- [선택적인 TAP 출력](#optional-tap-output)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#optional-tap-output)`
By:
`(#선택적인-tap-출력)`

File: ko_KR/readme.md [hunk 11]
```diff
@@ -0,0 +1,971 @@
...
+- [깨끗한 스텍 기록](#clean-stack-traces)
```
Comment (forresst) [inline ko_KR/readme.md]: Bad link, replace:
`(#clean-stack-traces)`
By:
`(#깨끗한-스택-기록)`

File: ko_KR/readme.md [hunk 12]
```diff
@@ -0,0 +1,971 @@
...
+test(t => {
+	return somePromise().then(result => {
+		t.is(result, 'unicorn');
+	});
+});
+```
+
+### 생성기 함수 지원
+
+AVA는 내부적으로 [생성기 함수](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)를 지원합니다.
```
Comment (forresst) [inline ko_KR/readme.md]: korean link is better, replace:
`(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function*)`
by:
`(https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/function*)`

File: ko_KR/readme.md [hunk 13]
```diff
@@ -0,0 +1,971 @@
...
+
+# ![AVA](https://github.com/sindresorhus/ava/blob/master/media/header.png)
+
+> 미래적인 테스트 러너
+
+[![Build Status: Linux](https://travis-ci.org/sindresorhus/ava.svg?branch=master)](https://travis-ci.org/sindresorhus/ava) [![Build status: Windows](https://ci.appveyor.com/api/projects/status/igogxrcmhhm085co/branch/master?svg=true)](https://ci.appveyor.com/project/sindresorhus/ava/branch/master) [![Coverage Status](https://coveralls.io/repos/sindresorhus/ava/badge.svg?branch=master&service=github)](https://coveralls.io/github/sindresorhus/ava?branch=master) [![Gitter](https://badges.gitter.im/join_chat.svg)](https://gitter.im/sindresorhus/ava)
+
+비록 JavaScript가 싱글-스레드 기반으로 작동할지라도, Node.js에서의 입출력은 비동기 생태계를 통해 병렬 처리가 가능합니다. AVA는 무거운 입출력 작업에 특히 유용한 이 장점을 적극적으로 활용하여 테스트를 동시에 처리합니다. 또한, 테스트 파일들은 분리된 프로세스처럼 병렬로 실행되므로 더 좋은 성능과 각 테스트 파일에 대한 고립된 환경을 제공합니다. Mocha에서 AVA로 [이전](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0)한 Pageres는 테스트 시간을 31초에서 11초로 줄였습니다. 테스트를 동시에 실행하는 능력을 갖추는 것은 더 작은 테스트를 작성하도록 유도합니다. 이 말은 테스트가 더 이상 전역 상태나 다른 테스트의 상태에 의존하지 않는다는 것입니다. 정말 대단하지 않나요?
+
+*기여하는 방법에 대해 찾아보고 있다면 (이슈/PR/etc), 우리의 [기여 가이드](contributing.md)를 확인하세요.*
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/contributing.md)`

File: ko_KR/readme.md [hunk 14]
```diff
@@ -0,0 +1,971 @@
...
+$ npm test
+```
+
+### 감시 모드로 실행하기
+
+```console
+$ npm test -- --watch
+```
+
+AVA는 지능형 감시 모드와 함께 제공됩니다. [이 레시피에 대해 자세히 알아보기](docs/recipes/watch-mode.md).
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/watch-mode.md)`

File: ko_KR/readme.md [hunk 15]
```diff
@@ -0,0 +1,971 @@
...
+			"stage-0",
+			"react"
+		]
+	},
+	"ava": {
+		"babel": "inherit"
+	},
+}
+```
+더 많은 예시와 자세한 구성 옵션에 대한 설명은 AVA의 [`.babelrc` 레시피](docs/recipes/babelrc.md)를 참고하세요.
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md)`

File: ko_KR/readme.md [hunk 16]
```diff
@@ -0,0 +1,971 @@
...
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
+- [`t.plan()`를 언제 사용할까](docs/recipes/when-to-use-plan.md)
+- [브라우저 테스팅](docs/recipes/browser-testing.md)
+- [TypeScript](docs/recipes/typescript.md)
+- [Babel 구성](docs/recipes/babelrc.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md)`

File: ko_KR/readme.md [hunk 17]
```diff
@@ -0,0 +1,971 @@
...
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
+- [`t.plan()`를 언제 사용할까](docs/recipes/when-to-use-plan.md)
+- [브라우저 테스팅](docs/recipes/browser-testing.md)
+- [TypeScript](docs/recipes/typescript.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/typescript.md)`

File: ko_KR/readme.md [hunk 18]
```diff
@@ -0,0 +1,971 @@
...
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
+- [`t.plan()`를 언제 사용할까](docs/recipes/when-to-use-plan.md)
+- [브라우저 테스팅](docs/recipes/browser-testing.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/browser-testing.md)`

File: ko_KR/readme.md [hunk 19]
```diff
@@ -0,0 +1,971 @@
...
+### 동시성과 병행성은 무슨 차이가 있나요?
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
+- [`t.plan()`를 언제 사용할까](docs/recipes/when-to-use-plan.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/when-to-use-plan.md)`

File: ko_KR/readme.md [hunk 20]
```diff
@@ -0,0 +1,971 @@
...
+
+### 동시성과 병행성은 무슨 차이가 있나요?
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
+- [엔드 포인트 테스팅](docs/recipes/endpoint-testing.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/endpoint-testing.md)`

File: ko_KR/readme.md [hunk 21]
```diff
@@ -0,0 +1,971 @@
...
+[안드로메다 은하](https://ko.wikipedia.org/wiki/%EC%95%88%EB%93%9C%EB%A1%9C%EB%A9%94%EB%8B%A4_%EC%9D%80%ED%95%98)입니다.
+
+### 동시성과 병행성은 무슨 차이가 있나요?
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
+- [감시 모드](docs/recipes/watch-mode.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/watch-mode.md)`

File: ko_KR/readme.md [hunk 22]
```diff
@@ -0,0 +1,971 @@
...
+
+[안드로메다 은하](https://ko.wikipedia.org/wiki/%EC%95%88%EB%93%9C%EB%A1%9C%EB%A9%94%EB%8B%A4_%EC%9D%80%ED%95%98)입니다.
+
+### 동시성과 병행성은 무슨 차이가 있나요?
+
+[동시성은 병행성이 아닙니다. 동시성은 병행성이 가능하도록 만듭니다.](https://stackoverflow.com/q/1050222)
+
+## 레시피
+
+- [코드 커버리지](docs/recipes/code-coverage.md)
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/code-coverage.md)`

File: ko_KR/readme.md [hunk 23]
```diff
@@ -0,0 +1,971 @@
...
+		]
+	},
+	"ava": {
+		"babel": "inherit"
+	},
+}
+```
+더 많은 예시와 자세한 구성 옵션에 대한 설명은 AVA의 [`.babelrc` 레시피](docs/recipes/babelrc.md)를 참고하세요.
+
+참고로 AVA는 구성에 상관없이 *언제나* [내부적으로 몇 가지 플러그인](docs/recipes/babelrc.md#notes)을 적용합니다. 하지만 실제 코드에 영향을 줄 가능성은 적습니다.
```
Comment (forresst) [inline ko_KR/readme.md]: Please note this link does not exist yet in Korean, it's best to link to the English version : `(https://github.com/sindresorhus/ava/blob/master/docs/recipes/babelrc.md#notes)`

**Comment by forresst** [comment]
@preco21  Except my remarks, it's beautiful work! Thank you.

Warn me when you have corrected the errors

**Comment by preco21** [comment]
@forresst Of course! Thank you. I will work on it.

**Comment by preco21** [comment]
@forresst I have updated the remarks! Please review again.

File: ko_KR/readme.md [hunk 24]
```diff
@@ -0,0 +1,971 @@
+___
+**역주**
+
+이 문서는 [readme.md](https://github.com/sindresorhus/ava/blob/master/readme.md)의 한국어 번역입니다. [이곳](https://github.com/sindresorhus/ava/commits/71404c23302d825095659c70cb9a1b08251697ad...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9)에서 AVA의 master 브랜치와 이 문서의 차이를 확인할 수 있습니다. (만약 차이가 없다면 문서가 최신 버전임을 의미합니다)
```
Comment (forresst) [inline ko_KR/readme.md]: Oups, bad link, sorry, replace:
`https://github.com/sindresorhus/ava/commits/71404c23302d825095659c70cb9a1b08251697ad...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9`
by:
`https://github.com/sindresorhus/ava/compare/71404c23302d825095659c70cb9a1b08251697ad...master#diff-0730bb7c2e8f9ea2438b52e419dd86c9`

**Comment by forresst** [comment]
@preco21 Except one bad link (sorry) It's perfect. LGTM

@sindresorhus it's good for you ?

**Comment by preco21** [comment]
@forresst Okay, I have updated!

**Comment by forresst** [comment]
LGTM cc @sindresorhus

**Comment by sindresorhus** [comment]
Thank you @preco21 :)

I've added you as a collaborator to this repo. Feel free to commit docs updates directly.

**Comment by jamestalmage** [comment]
⭐ 🌟 🌟 ⭐ 

Thank you!!!

⭐ 🌟 🌟 ⭐

**Comment by forresst** [comment]
Thank you @preco21 :tada:

**Pull request merged**